src/main/scala/lc/core/config.scala

```diff
@@ -49,6 +49,13 @@ object Config {
   val allowedMedia: Set[String] = captchaConfig.flatMap(_.allowedMedia).toSet
   val allowedInputType: Set[String] = captchaConfig.flatMap(_.allowedInputType).toSet
 
+  private val random = new scala.util.Random(seed)
+
+  def getNextRandomInt(max: Int): Int =
+  random.synchronized {
+    random.nextInt(max)
+  }
+
```

Comment:
The HelperFunctions class also has a random number generator. Can we use one common source?

Reply:
Which one do we go ahead with?

Reply:
The random number generator needs to be accessible in Java also. So putting it in HelperFunctions.java makes more sense.

However, the seed is configurable. So, the framework needs to apply the seed from the Config to the generator in HelperFunctions.

Suggestion:
1. Instantiate a random generator in HelperFunctions
2. Set the seed of this generator on framework startup
3. The random functions in `HelperFunctions` should use the common generator with synchronisation.
4. The  function Config.getNextRandomInt() can be moved to HelperFunctions, and it should use the common generator.